README.md

```diff
@@ -18,25 +18,26 @@ If you finish early, try tackling the [Moonrise Kingdom exercise](https://github
 
 Open up `index.html` in a text editor and browser. Link to the provided stylesheet.
 
-> Commit
+
 
 ### #2
 
 Identify one non-semantic tag and replace it with a semantic one.
 
-> Commit
+<div class= <main>   </main>
```

Comment:
:+1: although to be clear, you would replace `<div class="main"></div>` with `<main></main>`